subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -547,6 +547,13 @@ config BT_RX_USER_PDU_LEN
 	  header and payload. It does not account for HCI event headers as these
 	  PDUs are assumed to not go across HCI.
 
+config BT_CTLR_USER_CPR_INTERVAL
```

Comment:
Well it was supposed to be a feature flag for user defined CPR intervals not only the minimum limit, but we have only implemented the minimum limit. So that we don't end up with a BT_CTLR_USER_CPR_INTERVAL_MAX also in the future.

Reply:
I would prefer to have the name of the feature flag and its implementation in agreement. We can drop the `_MIN` suffix if we implement a maximum limit in the future.

Reply:
@cvinayak Any comments regarding this change?

Reply:
I will go with @wopu-ot  and add the `_MIN` suffix, saves us from future rename if we want to set max value.

Reply:
Done